pkg/arvo/app/acme.hoon

```diff
@@ -344,7 +344,7 @@
 ::
 =/  directory-base=purl
   =-  (need (de-purl:html -))
-  'https://acme-v02.api.letsencrypt.org/directory'
+  'https://acme-staging-v02.api.letsencrypt.org/directory'
```

Comment:
Guessing this staging URL might have slipped in unnoticed.

Reply:
yep, fixed.